spec/fixtures/unit/data_providers/environments/hiera_bad_syntax_hocon/data/bad.hocon

```diff
@@ -0,0 +1 @@
+"test::param_a": "env data param_a is 10":
```

Comment:
Is there any meaning to the fact that this file is named `bad`, or is the file name just arbitrary?